wgsl/index.bs

```diff
@@ -3805,8 +3805,106 @@ shorten vector declarations based on the type or number of elements provided.
 If you want `vec4<f32>` you must provide 4 float values in the constructor.
 
 ## Precedence ## {#precedence}
-
-Issue: (dsinclair) Write out precedence rules. Matches c and glsl rules ....
+<table class='data'>
+  <thead>
+    <tr>
+      <td>Precedence
+      <td>Operators
+      <td>Description
+      <td>Associativity
+  </thead>
+  <tr>
+    <td rowspan='1' style='text-align: left'><b>1 (highest)</b>
```

Comment:
If you use `<th>` for the first column instead of `<td>`, then you won't need the `<b>`.

Reply:
Done